helpers/general/getClassNames.ts

```diff
@@ -0,0 +1,16 @@
+import { TypeClassNames } from '@/types/index'
+
+type TypegetClassNamesProps = TypeClassNames
+
+type TypeGetClassNamesOutput = {
+  readonly [key: string]: boolean
+}
+
+const getClassNames = ({
+  classNames = []
+}: TypegetClassNamesProps): TypeGetClassNamesOutput =>
```

Comment:
should follow camel case pattern
```suggestion
}: TypeGetClassNamesProps): TypeGetClassNamesOutput =>
```